packages/validator/src/services/distributedAttestation.ts

```diff
@@ -0,0 +1,63 @@
+import {toHexString} from "@chainsafe/ssz";
+import {computeEpochAtSlot} from "@lodestar/state-transition";
+import {phase0, Slot} from "@lodestar/types";
+import {sleep} from "@lodestar/utils";
+import {AttestationService, AttestationServiceArgs} from "./attestation.js";
+import {groupAttDutiesByCommitteeIndex} from "./utils.js";
+
+export class DistributedAttestationService extends AttestationService {
+  constructor(...args: AttestationServiceArgs) {
+    super(...args);
+  }
+
+  protected runAttestationTasks = async (slot: Slot, signal: AbortSignal): Promise<void> => {
```

Comment:
the logic implemented here will further deviate from normal attestation service once we will support [attestation committee aggregation selection](https://docs.google.com/document/d/1q9jOTPcYQa-3L8luRvQJ-M0eegtba4Nmon3dpO79TMk/mobilebasic)